mono/tests/microbenchmarks/Makefile.am

```diff
@@ -0,0 +1,72 @@
+include $(top_srcdir)/mk/common.mk
+
+BENCHMARKDOTNET_PATH=BenchmarkDotNet
+
+SUBMODULES_CONFIG_FILE = $(top_srcdir)/mono/tests/microbenchmarks/SUBMODULES.json
+include $(top_srcdir)/scripts/submodules/versions.mk
+
+$(eval $(call ValidateVersionTemplate,BenchmarkDotNet,BENCHMARKDOTNET))
+
+# Bump the given submodule to the revision given by the REV make variable
+# If COMMIT is 1, commit the change
+bump-BenchmarkDotNet: __bump-version-BenchmarkDotNet
+
+# Bump the given submodule to the branch given by the BRANCH/REMOTE_BRANCH make variables
+# If COMMIT is 1, commit the change
+bump-branch-BenchmarkDotNet: __bump-branch-BenchmarkDotNet
+
+# Bump the given submodule to its current GIT version
+# If COMMIT is 1, commit the change
+bump-current-BenchmarkDotNet: __bump-current-version-BenchmarkDotNet
+
+clean-local:
+	$(RM) -r $(BENCH_PATH)
+
+EXTRA_DIST=SUBMODULES.json
+
+BenchmarkDotNet.stamp: Makefile 
+	$(MAKE) reset-BenchmarkDotNet
+	$(MAKE) BenchmarkDotNet/src/BenchmarkDotNet/bin/Release/net46/BenchmarkDotNet.dll
+	touch $@
+
+BenchmarkDotNet/src/BenchmarkDotNet/bin/Release/net46/BenchmarkDotNet.dll:
+	cd BenchmarkDotNet && chmod +x ./build.sh && ./build.sh --skiptests=true
```

Comment:
We don't, it's built only when the stamp has to be remade. This is the pattern we use for llvm, btls, jemalloc, and now for BenchmarkDotNet. I agree that it would be great to depend on binary artifacts instead. How would you like to see me do that?